src/pages/index.js

```diff
@@ -46,26 +49,45 @@ const handleCardClick = (cardData) => {
 };
 
 const createCard = (cardData) => {
-  const card = new Card(cardData, "#card-template", handleCardClick);
+  const card = new Card(cardData, "#card-template", handleCardClick, api);
   return card.createCardElement();
 };
 
-const section = new Section(
-  {
-    items: initialCards,
-    renderer: (item) => {
-      const cardElement = createCard(item);
-      section.addItem(cardElement);
-    },
+const api = new Api({
+  baseUrl: "https://around.nomoreparties.co/v1/cohort-3-en",
+  headers: {
+    authorization: "35337f3b-35e8-4dc0-a9b5-b6c4dd4127c3",
+    "Content-Type": "application/json",
   },
-  ".gallery__cards"
-);
+});
+
+let cardList;
+
+api.getInitialCards().then((result) => {
```

Comment:
You can receive all the initial data together using `Promise.all`, so you will save user's id and provide it to Card with constructor arguments and will be able to define if user is card's owner

```js
let userId;

Promise.all([api.getUserInfo(), api.getInitialCards()]).then(([user, cards]) => {
  userInfo.setProfileInfo(userData.name, userData.about);
  userId = user._id;  // set user id
  // initialize cards section
}).catch(() => console.log(...))
```